core/src/main/java/com/graphhopper/routing/util/MotorcycleFlagEncoder.java

```diff
@@ -93,30 +100,37 @@ public MotorcycleFlagEncoder( int speedBits, double speedFactor, int maxTurnCost
 
         // autobahn
         defaultSpeedMap.put("motorway", 100);
-        defaultSpeedMap.put("motorway_link", 70);
+        defaultSpeedMap.put("motorway_link", 80);
         defaultSpeedMap.put("motorroad", 90);
         // bundesstraße
         defaultSpeedMap.put("trunk", 80);
         defaultSpeedMap.put("trunk_link", 75);
         // linking bigger town
-        defaultSpeedMap.put("primary", 65);
-        defaultSpeedMap.put("primary_link", 60);
+        defaultSpeedMap.put("primary", 100);
+        defaultSpeedMap.put("primary_link", 80);
         // linking towns + villages
-        defaultSpeedMap.put("secondary", 60);
-        defaultSpeedMap.put("secondary_link", 50);
+        defaultSpeedMap.put("secondary", 90);
+        defaultSpeedMap.put("secondary_link", 70);
         // streets without middle line separation
-        defaultSpeedMap.put("tertiary", 50);
-        defaultSpeedMap.put("tertiary_link", 40);
-        defaultSpeedMap.put("unclassified", 30);
+        defaultSpeedMap.put("tertiary", 80);
+        defaultSpeedMap.put("tertiary_link", 60);
+        defaultSpeedMap.put("unclassified", 40);
         defaultSpeedMap.put("residential", 30);
         // spielstraße
         defaultSpeedMap.put("living_street", 5);
         defaultSpeedMap.put("service", 20);
         // unknown road
-        defaultSpeedMap.put("road", 20);
+        defaultSpeedMap.put("road", 40);
         // forestry stuff
         defaultSpeedMap.put("track", 15);
 
+        enjoyableTags.add("scenic");
+
+        // List is typcially set in urban areas
+        frustratingTags.add("lit");
```

Comment:
Exactly that was a quick and probably dirty fix, but the results work quite well, at least for my area. But you are right if #230 is fixed we should use these results. I hope you are fine if we leave this here until #230 is ready?

But in general lit roads are a sign of high traffic volume and we don't want traffic.